oauth2client/contrib/django_util/__init__.py

```diff
@@ -299,7 +389,10 @@ def has_credentials(self):
     @property
     def credentials(self):
         """Gets the authorized credentials for this flow, if they exist"""
-        return storage.get_storage(self.request).get()
+        try:
+            return get_storage(self.request).get()
+        except storage.DjangoORMError:
```

Comment:
Intentionally silently swallowed due to cases where you try to get credentials on an anonymous user. I think part of the motivation was trying to minimize changes to existing Storage method but given this is a big change should prob just change things all at once. Let me revisit this.